Microsoft.Identity.Web/Microsoft.Identity.Web.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="2.1.1" />
```

Comment:
I moved the ASP.NET ref to 2.2.0 now
you might want to resync the one from the [Web App tutorial](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/tree/tutorials/Microsoft.Identity.Web)

this still evolves a bit (in particular doc), but should stabilize soon.

Question: how do we ensure the implementation in your web samples don't diverge too much?
Would it make sense to have your implementation into the Web App tutorial (I would think so) instead of using the REST API there?